tests/test_models.py

```diff
@@ -1,23 +1,24 @@
-from app.model import *
+from .base import BaseTest
+from app.models import *
 import sys
 sys.path.append("..")
 
 
-class TestClassModel(object):
+class TestClassModel(BaseTest):
     def test_languages(self):
         """basic test Languages Collection"""
         lang = Languages(language="russian", code="RUS",
                          type="foreign", default=False)
         lang.save()
-        assert Languages.objects.count() > 0
+        assert Languages.objects.first().language == "russian"
```

Comment:
Also check that russian only has one entry and is the only element in the database.
`Languages.objects.count()==1`

Reply:
Will do